src/pages/components/tile/style.mdx

```diff
@@ -9,6 +9,40 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 import { Checkmark } from '@carbon/icons-react';
 import { white } from '@carbon/themes';
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+and structure.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>AI presence</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+<PageDescription>
```

Comment:
Page description and anchor links are repeated here. Remove the duplications.

Reply:
Ohh damn, idk how it happended lol but good catch thanks!